packages/hash/api/codegen.yml

```diff
@@ -17,8 +17,6 @@ generates:
       - "typescript-resolvers"
     config:
       noSchemaStitching: true
-      mappers:
-        Page: ../types/dbTypes#DbPage
```

Comment:
@nathggns @kachkaev any idea what this was for and if we need a replacement?

Reply:
I think we don't need one, because there's already a proper definition for the `Page` type in graphql. I don't really understand why it was importing the page type from `dbType` though. I suspect this was maybe added at first as a shortcut and is now unnecessary.

I'm not sure though, if anyone knows more to explain it would be useful